raiden_contracts/tests/fixtures/channel.py

```diff
@@ -21,13 +21,14 @@
 
 
 @pytest.fixture(scope='session')
-def create_channel(token_network):
+def create_channel(token_network, web3, call_and_transact):
     def get(A, B, settle_timeout=TEST_SETTLE_TIMEOUT_MIN):
         # Make sure there is no channel existent on chain
         assert token_network.functions.getChannelIdentifier(A, B).call() == 0
 
         # Open the channel and retrieve the channel identifier
-        txn_hash = token_network.functions.openChannel(A, B, settle_timeout).transact()
+        txn_hash = token_network.functions.openChannel(A, B, settle_timeout).call_and_transact()
+        web3.testing.mine(1)
```

Comment:
Doesn't transact cause eth_tester to mine a block? (There seems to be a need for an extra mine() in multiple places. I only ask once because I assume your answer will cover the other cases, too)

Reply:
I don't know the reason, but `call_and_transact()` doesn't mine a block.

Perhaps I should add a test that calls `call_and_transact()` and checks the change of the block numbers.

Reply:
The test-writing is deferred to https://github.com/raiden-network/raiden-contracts/issues/820